### PR TITLE
Various production improvements

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Add consensus to cloud latency graph to importer dashboard
- Add consensus to download latency graph to importer dashboard
- Add Spring configuration properties metadata for autocomplete & documentation when editing `application.yml`
- Bump chart dependencies
- Bump image tags for Prometheus, PostgreSQL, Redis, TimescaleDB
- Change monitor and importer alert thresholds to reflect mainnet
- Change monitor auto-creation of entities to always set `adminKey` to operator
- Change monitor timeouts and switch from records to receipts to reduce errors in mainnet
- Change REST API page limits from 1000 to 500 to improve performance
- Disable kube proxy monitoring since it doesn't work
- Enable kube API server monitoring
- Fix cAdvisor metrics missing by re-enabling kubelet monitoring
- Fix Grafana not working with multiple clusters and a single domain name
- Fix Grafana not working with port forwarding
- Fix monitor dashboard showing zero for subscriber rate
- Fix REST API dashboard showing the sum of resources for all pods
- Fix REST API database connection pool not shutting down gracefully
- Fix TimescaleDB PodMonitor always being created
- Improve REST API performance at higher load by bumping resource limits & max replicas
- Improve Traefik performance at higher load by bumping resource limits

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
Sorry for the hodgepodge of changes.

IDE autocomplete in action:

Screen Shot 2021-05-27 at 22 08 05

**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/health.js [hunk 1]
```diff
@@ -54,14 +56,17 @@ const livenessCheck = async () => {};
...
-  });
-}
+const beforeShutdown = async () => {
+  logger.info(`Closing connection pool`);
+  return pool.end().then(
+    new Promise((resolve) => {
+      setTimeout(resolve, config.shutdown.timeout);
+    })
+  );
+};
```
Comment (xin-hedera) [inline hedera-mirror-rest/health.js:66]: I don't think we need the delay by `setTimeout` at all. `terminus` doc highlighted the use case that for  load-balancers (like the nginx ingress controller)  other than the k8s services, the load balancer may not remove the pod even though it's signaled to shutdown, so the delay here (longer than the readiness probes interval) is needed for the load balancer to mark the pod as not ready.

The reason I bring it up is, the promise chaining won't work for the purpose that after the pool ends, delay `config.shutdown.timeout`.

So if we don't need the delay, we can simply do `return pool.end()` here.

on the other hand, to make the delay work, we have to sth like

```javascript
return pool.end().then(
  () => new Promise((resolve) => setTimeout(resolve, config.shutdown.timeout)),
  () => new Promise((resolve) => setTimeout(resolve, config.shutdown.timeout))
);
```

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/health.js (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-rest/health.js:66]: I'm not 100% sure, but I think Traefik just uses the kubernetes services to get the endpoints and then route directly to pods. It has its own internal load balancing schemes.

Comment (steven-sheehy) [reply hedera-mirror-rest/health.js:66]: But then if that pod does go unready then the endpoints will be updated and Traefik should see that and stop routing new connections to it. So think you're right and we can remove the timeout.

Comment (steven-sheehy) [reply hedera-mirror-rest/health.js]: Done

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by ijungmann — APPROVED** [review]

**Pull request merged**